code/modules/mob/living/simple_animal/bot/construction.dm

```diff
@@ -390,11 +390,19 @@
 			if(istype(I, /obj/item/melee/baton))
 				if(!can_finish_build(I, user))
 					return
-				to_chat(user, "<span class='notice'>You complete the Securitron! Beep boop.</span>")
-				var/mob/living/simple_animal/bot/secbot/S = new(Tsec)
-				S.name = created_name
-				S.baton_type = I.type
-				S.robot_arm = robot_arm
+				if(prob(95))//monkestation edit
+					to_chat(user, "<span class='notice'>You complete the Securitron! Beep boop.</span>")
+					var/mob/living/simple_animal/bot/secbot/S = new(Tsec)
+					S.name = created_name
+					S.baton_type = I.type
+					S.robot_arm = robot_arm
+				else
+					to_chat(user, "<span class='notice'>You complete the... Shitcuritron? Are you sure you did that right?</span>")
+					var/mob/living/simple_animal/bot/secbot/pizzky/S = new(Tsec)
+					S.name = created_name
+					S.baton_type = I.type
+					S.robot_arm = robot_arm
```

Comment:
Ton of copy-paste. Just put the to_chat in the if/else block nad move everything else out

Reply:
That would not work, Pizzky is a separate mob entirely and couldn't be defined before the check

Reply:
could do something like this:
```dm
var/mob/living/simple_animal/bot/secbot/S
if(prob(95))//monkestation edit
	to_chat(user, "<span class='notice'>You complete the Securitron! Beep boop.</span>")
	S = new(Tsec)
else
	to_chat(user, "<span class='notice'>You complete the... Shitcuritron? Are you sure you did that right?</span>")
	S = new mob/living/simple_animal/bot/secbot/pizzky(Tsec)
S.name = created_name
S.baton_type = I.type
S.robot_arm = robot_arm